app/models/work.rb

```diff
@@ -608,6 +608,9 @@ def default_date
 
   # Virtual attribute for series
   def series_attributes=(attributes)
+    if self.preview_mode
```

Comment:
I think that the series isn't supposed to be saved immediately in two cases: (1) if the user presses "Preview", or (2) if the user has entered invalid data elsewhere on the form and pressed "Post."

In other words, if the user is bounced back to the edit form with a message saying, "Sorry! We couldn't save this work," then I don't think the changes made to the work's series should be saved. But that wasn't mentioned in the description of the issue, so maybe I'm misremembering the desired behavior?

If that is the desired behavior, though, this approach won't work, and you might want to play around with calling `serial_works.build(series: old_series)` instead of `self.series << old_series`.

Reply:
Didn't think about saving errors case. Changed implementation so that work-series relationship is not being saved in both cases. Thanks for the tip!